app/tests/test-query.py

```diff
@@ -0,0 +1,29 @@
+import asyncio
+
+from modules.database.database import Postgres
+
+pg_test = Postgres()
+
+query = """
+    insert into test.time (
+        datetime,
+        session_id,
+        message_id,
+        node_name,
+        exec_time
+    ) values (
+        '2025-01-08 12:00:00',
+        'test',
+        'test',
+        'test',
+        1
+    );
```

Comment:
```suggestion
    INSERT INTO test.time (
        datetime,
        session_id,
        message_id,
        node_name,
        exec_time
    ) VALUES (
        '2025-01-08 12:00:00',
        'test',
        'test',
        'test',
        1
    );
```
Being nitpicky, but just for convention, maybe better to be like this ㅎㅎ